keylime-agent/src/main.rs

```diff
@@ -295,7 +295,14 @@ async fn main() -> Result<()> {
     // ownership of TPM access, which will not be implemented here.
     let tpm_ownerpassword = &config.agent.tpm_ownerpassword;
     if !tpm_ownerpassword.is_empty() {
-        let auth = Auth::try_from(tpm_ownerpassword.as_bytes())?;
+        let auth = if tpm_ownerpassword.starts_with("hex:") {
+            let (_, hex_ownerpassword) = tpm_ownerpassword.split_at(4);
+            let decoded_ownerpassword = hex::decode(hex_ownerpassword)
+                .expect("Failed to decode tpm_ownerpassword hex");
```

Comment:
expect works similarly to unwrap. Can you change this to an error that is returned?

Reply:
Yes I have now changed this, thanks!